integration-tests/ledger-signer.spec.ts

```diff
@@ -36,15 +36,15 @@ CONFIGS().forEach(({ lib, setup, rpc }) => {
       done();
     });
 
-    it('LedgerSigner is instantiable with default parameters', () => {
+    it('As a User I want to verify that LedgerSigner is instantiable with default parameters', () => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"

Reply:
@michaelkernaghan

Reply:
changed to : Verify signing and injecting a transaction with Ledger